src/Components/Carousel/Carousel.js

```diff
@@ -2,14 +2,27 @@ import React, {useEffect, useState} from 'react';
 import './Carousel.scss';
 
```

Comment:
Remove empty line here. Let's keep import statements grouped at the top of the file without adding new line between them.